helm-charts/seldon-core-operator/values.yaml

```diff
@@ -206,4 +206,4 @@ engine:
 # Explainer image
 explainer:
   image: seldonio/alibiexplainer:1.12.0-dev
-  image_v2: seldonio/mlserver:0.6.0.dev4
+  image_v2: seldonio/mlserver:1.0.0.rc1-alibi-explain
```

Comment:
I havent tested this specifically with just `alibi-explain` runtime but hopefully the integration test will run the tests needed to confirm that.